packages/app/src/app/globals.css

```
@@ -0,0 +1,2646 @@
+}
+
+.module-workspace-slot.is-active .module-actions {
+  animation: workspace-control-enter 440ms var(--easing-mechanical) 500ms both;
+}
+
+.module-workspace-slot.is-active .module-readout {
+  animation: workspace-control-enter 440ms var(--easing-mechanical) 570ms both;
+}
+
+@keyframes workspace-control-enter {
+  from {
```

Stylelint (17.14.0)

[error] Expected "from" to be "0%" (keyframe-selector-notation)

(keyframe-selector-notation)